### PR TITLE
Vcpkg features

### PR DESCRIPTION
**Pull request by petersteneteg**
Vcpkg: added dicom feature for the modules dicom/gdcm module. Also made ffmpeg hdf5 info features that are on by default. turn off by setting VCPKG_MANIFEST_NO_DEFAULT_FEATURES to ON

### CODE REVIEW

File: vcpkg.json [hunk 1]
```diff
@@ -84,9 +93,13 @@
         }
       ]
     },
+    "dicom" : {
+      "description": "Grassroots DICOM library",
```
Comment (ResearchDaniel) [inline vcpkg.json:97]:
```suggestion
      "description": "Support DICOM (.dcm) files, see https://github.com/inviwo/modules/tree/master/medvis/dicom",
```

File: vcpkg.json [hunk 2]
```diff
@@ -47,11 +40,27 @@
     "zlib",
     "pegtl",
     "inja",
-    "ffmpeg",
     "warn",
     "zlib"
   ],
+  "default-features": [
+    "hdf5",
```
Comment (ResearchDaniel) [inline vcpkg.json:47]: The HDF5 module is not on by default, so it should not be added here?

**Review by ResearchDaniel — DISMISSED** [review]

**Review by martinfalk — APPROVED** [review]

File: vcpkg.json (hunk 2 above)
Comment (petersteneteg) [reply vcpkg.json:47]: It used to be on, I will keep it on

File: vcpkg.json (hunk 1 above)
Comment (petersteneteg) [reply vcpkg.json:97]: I want to keep this short.

**Pull request merged**